ketcher-autotests/tests/Reaction-tool/Mapping-Unmapping tool/reaction-tools.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { test } from '@playwright/test';
+import {
+  takeLeftToolbarScreenshot,
+  LeftPanelButton,
+  selectLeftPanelButton,
+  selectNestedTool,
+  ReactionMappingTool,
+  ArrowTool,
+  delay,
+  DELAY_IN_SECONDS,
+  openFileAndAddToCanvas,
+} from '@utils';
+
+test.describe('Open Ketcher', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+    await page.waitForSelector('.Ketcher-root');
```

Comment:
What does this line do? May be we can create a separate function for that? And/or we need to add test-id